src/stm32/stm32f0_i2c.c

```diff
@@ -27,6 +27,9 @@ struct i2c_info {
   DECL_CONSTANT_STR("BUS_PINS_i2c1", "PB6,PB7");
   DECL_ENUMERATION("i2c_bus", "i2c1a", 1);
   DECL_CONSTANT_STR("BUS_PINS_i2c1a", "PF1,PF0");
+#elif CONFIG_MACH_STM32F7
+  DECL_ENUMERATION("i2c_bus", "i2c1_PB6_PB7", 0);
```

Comment:
did not test i2c to work on the hardware I got.
only added minmum config to match remram pinout